### PR TITLE
nodePackages: init he and escape-string-regexp

### PR DESCRIPTION
**Pull request by expipiplus1**
These are a prerequisite for building the [Vulkan
documentation](https://github.com/KhronosGroup/Vulkan-Docs). Vulkan-Docs
doesn't come with a well-formed package.json which makes it difficult to
use with node2nix or other tools. Having these packages here makes it
very easy.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by prusnak** [comment]
Please rework your PR. It now has a merge conflict after PR https://github.com/NixOS/nixpkgs/pull/89184 has been merged

**Comment by expipiplus1** [comment]
@prusnak I have rebased on master.

File: pkgs/development/node-packages/node-packages.nix [hunk 1]
```diff
@@ -63212,6 +63230,24 @@ in
...
+    packageName = "he";
+    version = "1.2.0";
+    src = fetchurl {
+      url = "https://registry.npmjs.org/he/-/he-1.2.0.tgz";
+      sha512 = "F/1DnUGPopORZi0ni+CvrCgHQ5FyEAHRLSApuYWMmrbSwoN2Mn/7k+Gl38gJnR7yyDZk6WLXwiGod1JOWNDKGw==";
+    };
+    buildInputs = globalBuildInputs;
+    meta = {
+      description = "A robust HTML entities encoder/decoder with full Unicode support.";
+      homepage = https://mths.be/he;
```
Comment (prusnak) [inline pkgs/development/node-packages/node-packages.nix:63244]: Does this come from node2nix? As per RFC45 the URLs should be quoted:

```suggestion
      homepage = "https://mths.be/he";
```

**Review by prusnak — CHANGES_REQUESTED** [review]

File: pkgs/development/node-packages/node-packages.nix (hunk 1 above)
Comment (expipiplus1) [reply pkgs/development/node-packages/node-packages.nix:63244]: done, thanks

**Review by prusnak — APPROVED** [review]

**Comment by expipiplus1** [comment]
Is it possible to get this merged, please?

**Comment by veprbl** [comment]
Landed as 74ff4f65ca4b0fd9a88da1f0910cbd6eda62bad8 and d6c0115cee9507b1908e7e1b4ce08995e429611d

**Comment by expipiplus1** [comment]
Thanks!

**Pull request closed without merging**